admin/src/app/admin/navigation/navigation.component.html

```diff
@@ -1,38 +1,49 @@
-<mat-sidenav-container class="sidenav-container">
+<mat-sidenav-container class="sidenav-container"
+                       autosize>
   <mat-sidenav #drawer
-               class="sidenav"
+               [ngClass]="{'expanded-nav': isExpanded}"
                fixedInViewport="true"
                [attr.role]="(isHandset$ | async) ? 'dialog' : 'navigation'"
                [mode]="(isHandset$ | async) ? 'over' : 'side'"
                [opened]="!(isHandset$ | async)">
-    <mat-toolbar color="primary">
-      Menu
+    <mat-toolbar class="header-side-nav">
+      <img class="logo"
+           src="assets/icons/tanam-icon.svg"
+           height="25"
+           style="margin-right:10px" /> <b>{{isExpanded? 'Tanam':''}}</b>
     </mat-toolbar>
     <mat-nav-list>
       <app-navigation-list-item routerLink="/_/admin/dashboard"
                                 icon="dashboard"
-                                title="Dashboard"></app-navigation-list-item>
-
-      <h3 mat-subheader>Content</h3>
+                                title="{{isExpanded? 'Dashboard':''}}"></app-navigation-list-item>
+      <h3 mat-subheader
+          *ngIf="isExpanded">Content</h3>
+      <mat-divider *ngIf="!isExpanded"></mat-divider>
       <app-navigation-list-item *ngFor="let contentType of contentTypes$ | async"
                                 routerLink="/_/admin/types/{{contentType.id}}"
                                 icon="{{contentType.icon}}"
-                                title="{{contentType.title}}"></app-navigation-list-item>
+                                title="{{isExpanded? contentType.title:''}}"></app-navigation-list-item>
 
-      <h3 mat-subheader>Manage</h3>
+      <h3 mat-subheader
+          *ngIf="isExpanded">Manage</h3>
+      <mat-divider *ngIf="!isExpanded"></mat-divider>
       <app-navigation-list-item routerLink="/_/admin/media"
                                 icon="image"
-                                title="Media files"></app-navigation-list-item>
+                                title="{{isExpanded? 'Media files':''}}"></app-navigation-list-item>
       <app-navigation-list-item routerLink="/_/admin/types"
                                 icon="library_books"
-                                title="Content types"></app-navigation-list-item>
+                                title="{{isExpanded? 'Content types':''}}"></app-navigation-list-item>
       <app-navigation-list-item routerLink="/_/admin/themes"
                                 icon="web"
-                                title="Themes"></app-navigation-list-item>
+                                title="{{isExpanded? 'Themes':''}}"></app-navigation-list-item>
       <app-navigation-list-item routerLink="/_/admin/settings"
                                 icon="settings"
-                                title="Settings"></app-navigation-list-item>
+                                title="{{isExpanded? 'Settings':''}}"></app-navigation-list-item>
```

Comment:
Perhaps we can pass the `isExpanded` parameter into the `app-navigation-list-item` instead and let the logic of displaying/hiding the title be in there instead.

**navigation.component.html**
```html
<app-navigation-list-item routerLink="/_/admin/settings"
                          icon="settings"
                          isExpanded="{{isExpanded}}"
                          title="Settings"></app-navigation-list-item>
```

**navigation-list-item.component.html**
```html
<a mat-list-item routerLink="{{routerLink}}" routerLinkActive="active">
  <mat-icon mat-list-icon>{{icon}}</mat-icon>
  <span *ngIf="isExpanded">{{title}}</span>
</a>
```

Reply:
Fixed in commit https://github.com/oddbit/tanam/pull/34/commits/e35287bcb00a28137adb85c4221b7e403f47fdf5